lib/matplotlib/tests/test_afm.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.afm as afm
```

Comment:
This is the kind of practices that break static checkers such as pyflakes with very little benefit considering a forgotten import should break the test.